src/ui-presets/ads.js

```diff
@@ -24,17 +24,29 @@ import {PlaybackControls} from '../components/playback-controls';
 export function adsUI(props: any): ?React$Element<any> {
   if (useDefaultAdsUi(props)) {
     return (
-      <div className={style.adGuiWrapper}>
+      <div className={[style.adGuiWrapper, style.onTopAd].join(' ')}>
+        <KeyboardControl player={props.player} config={props.config} />
```

Comment:
verify we want this. as now the user can seek to the end using the keyboard

Reply:
i think we do as we have it in the other ads view as well (and he can't really skip the ad)